Internship2022TeamPatri/Internship2022TeamPatri/Base.lproj/Main.storyboard

```diff
@@ -1,44 +1,164 @@
+                            <constraint firstItem="dnt-N3-a0X" firstAttribute="top" secondItem="j3I-nr-U43" secondAttribute="bottom" constant="38" id="yFl-4p-rMI"/>
                         </constraints>
                     </view>
+                    <navigationItem key="navigationItem" id="doW-Qo-hrF"/>
+                    <connections>
+                        <outlet property="emailLabel" destination="Fsy-vv-Ee4" id="Y11-zE-dVP"/>
+                        <outlet property="emailTextField" destination="j3I-nr-U43" id="keW-Pz-d4j"/>
+                        <outlet property="forgotPasswordButton" destination="t5I-ef-nl5" id="a2e-Zk-NkE"/>
+                        <outlet property="logInLabel" destination="Psx-bl-9hq" id="ArI-Ir-bkd"/>
+                        <outlet property="loginButton" destination="WHp-mV-d9K" id="aBB-iA-Ufd"/>
+                        <outlet property="notRegisteredLabel" destination="frg-90-dGI" id="kX5-Wx-Q8M"/>
+                        <outlet property="passwordLabel" destination="EG9-Xd-69U" id="kJT-rO-KN5"/>
+                        <outlet property="passwordTextField" destination="dnt-N3-a0X" id="79T-J3-RcA"/>
+                        <outlet property="signupButton" destination="dvi-O2-XsG" id="JnB-md-n0z"/>
+                    </connections>
                 </viewController>
                 <placeholder placeholderIdentifier="IBFirstResponder" id="dkx-z0-nzr" sceneMemberID="firstResponder"/>
             </objects>
-            <point key="canvasLocation" x="35" y="98"/>
+            <point key="canvasLocation" x="944.85981308411215" y="97.192224622030238"/>
+        </scene>
+        <!--Navigation Controller-->
+        <scene sceneID="AQP-qb-un9">
+            <objects>
+                <navigationController automaticallyAdjustsScrollViewInsets="NO" navigationBarHidden="YES" id="bj1-Dx-hew" sceneMemberID="viewController">
+                    <toolbarItems/>
+                    <navigationBar key="navigationBar" contentMode="scaleToFill" id="uJ1-tJ-kR3">
+                        <autoresizingMask key="autoresizingMask"/>
+                    </navigationBar>
+                    <nil name="viewControllers"/>
+                    <connections>
+                        <segue destination="BYZ-38-t0r" kind="relationship" relationship="rootViewController" id="BSM-jY-eh6"/>
+                    </connections>
+                </navigationController>
+                <placeholder placeholderIdentifier="IBFirstResponder" id="DXb-JU-xtl" userLabel="First Responder" customClass="UIResponder" sceneMemberID="firstResponder"/>
+            </objects>
+            <point key="canvasLocation" x="34.782608695652179" y="97.767857142857139"/>
         </scene>
     </scenes>
-    <resources>
-        <systemColor name="systemBackgroundColor">
-            <color white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
-        </systemColor>
-    </resources>
```

Comment:
Please remove your previous work. This file (Main.storyboard) should be untouched since you moved your work to another file (LogIn.storyboard)